board/piksiv3/rootfs-overlay/usr/bin/dump_core.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+/usr/bin/logger "CORE DUMP!! executable=$1 pid=$2 time=$3"
+echo "CORE DUMP!! executable=$1 pid=$2 time=$3" | /usr/bin/sbp_log --error
+
+corefile="/tmp/cores/core-$1-$2-$3"
+touch $corefile
+
+while read line
```

Comment:
I'm not crazy about this reading the core data as text and appending line-by-line.  I'd do this:
```
exec cat >> $corefile
```